packages/shared/src/components/RichTextEditor/types.ts

```diff
@@ -4,16 +4,15 @@ import { PluginComposer } from '@frontify/fondue';
 import { DesignTokens } from '@frontify/fondue/dist/components/RichTextEditor/types';
 
 export type RichTextEditorProps = {
-    settingsId: string;
     isEditing: boolean;
-    setBlockSettings: (settings: Record<string, unknown>) => void;
     value?: string;
     placeholder?: string;
     designTokens?: DesignTokens;
     columns?: number;
     gap?: string;
     plugins?: PluginComposer;
     updateValueOnChange?: boolean;
+    onStore: (value: string) => void;
```

Comment:
what do you think about `onBlur` instead of onStore? onStore kind of makes an assumption about how the consumer will use this prop.

Reply:
You are right, I adjusted it.